<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FightDialog</class>
 <widget class="QDialog" name="FightDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="infoLabel">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>10</y>
     <width>221</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Time to test your reactions in a Teekkari reaction fight!</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="startButton">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>170</y>
     <width>161</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>Begin</string>
   </property>
  </widget>
  <widget class="QLabel" name="resultLabel">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>70</y>
     <width>331</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
